src/ol/style/Icon.js

```diff
@@ -514,32 +520,24 @@ class Icon extends ImageStyle {
    * Set the height of the icon in pixels.
    *
    * @param {number} height The height to set.
+   * @api
```

Comment:
Same as above.

Reply:
I added them because they are used in the example.  Also if a `width` or `height` set in the constructor is invalidated by calling `setScaling` they provide a means to reset them.

Reply:
Reading this, it feels to me like the change suggested here makes the API worse than it is already.

Let's stop this effort. I created #14606 to fix the underlying issue, without any changes that cause headache.

I'm also going to create a pull request that removes the use of `setWidth()` and `setHeight()` from the `icon-width.html` example.